device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/IotHubStatusCode.java

```diff
@@ -40,10 +105,10 @@ public static IotHubServiceException getConnectionStatusException(IotHubStatusCo
             case UNAUTHORIZED:
                 transportException = new UnauthorizedException(statusDescription);
                 break;
-            case TOO_MANY_DEVICES:
+            case QUOTA_EXCEEDED:
                 transportException = new TooManyDevicesException(statusDescription);
```

Comment:
`TOO_MANY_DEVICES` was renamed to `QUOTA_EXCEEDED` but the associated method is still `TooManyDevicesException()`.